med_io/get_pad_and_patch.py

```diff
@@ -163,7 +167,7 @@ def get_predict_patches_index(data_img, patch_size, overlap_rate=0.5, start=None
     assert (len(patch_size) == len(start) == len(overlap_rate) == dim)
     patch_size = [min(data_size[i], patch_size[i]) for i in range(dim)]
     if output_patch_size is None:
-        step = patch_size - np.round(overlap_rate * patch_size)
+        step = patch_size - np.round(overlap_rate * patch_size)*2
```

Comment:
why step size changed? Previous calc wrong?

Reply:
I put a *2 because prediction on PyTorch implementation had a *2, but I think it does not make much difference if the *2 is deleted. I deleted it, because it is the one in the master branch